docs/production-deployment/cloud/nexus/use-cases.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: use-cases
+slug: /cloud/nexus/use-cases
+title: Common Use Cases for Temporal Nexus
+description: Common use cases for Temporal Nexus
+sidebar_label: Use Cases
+tags:
+  - Temporal Cloud
+  - Nexus
+keywords:
+  - Nexus use cases
+  - Temporal Nexus onboarding
+  - setting up Nexus Endpoints
+  - temporal cloud
+  - nexus setup
+
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+## Share Workflows Across Namespaces
+
+Nexus is purpose-built to connect Temporal Applications within and across Namespaces.
```

Comment:
These are general use cases and I believe this used to be "Getting Started". Use cases as a keyword is something that's highly sought out by users. On the website, it's one of the most sought out pages. Is there another name we can say for this page? When people ask for Nexus use cases, is this what they have in mind? 

If so, I would add a short intro -- There are two main use cases for Temporal Nexus: Sharing Workflows Across Namespaces and Decompose a Monolithic Namespace

Reply:
added an intro/summary of Nexus use cases, and used the sections below to elaborate on those